firmware/controllers/actuators/vvt.cpp

```diff
@@ -58,6 +58,12 @@ expected<angle_t> VvtController::observePlant() {
 
 expected<angle_t> VvtController::getSetpoint() {
 	int rpm = Sensor::getOrZero(SensorType::Rpm);
+	bool enabled = rpm > engineConfiguration->vvtControlMinRpm
+			&& engine->rpmCalculator.getSecondsSinceEngineStart(getTimeNowNt()) > engineConfiguration->vvtActivationDelayMs / MS_PER_SECOND
+			 ;
+	if (!enabled)
```

Comment:
avoid no-bracket if statements https://dwheeler.com/essays/apple-goto-fail.html

Reply:
i wish gcc has an error option for that

https://wiki.sei.cmu.edu/confluence/plugins/servlet/mobile?contentId=87152259#content/view/87152259